src/components/Header.js

```diff
@@ -1,19 +1,62 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable jsx-a11y/alt-text */
-import React, { useContext } from 'react'
-import {ProductContxt} from '../App'
+import React, { useContext, useState ,createContext} from 'react'
+import {ProductContxt,CartProudectContext} from '../App'
 import '../style/Header.css' 
+import '../style/Cart.css'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faCartShopping } from "@fortawesome/free-solid-svg-icons"
-
+import { faShoppingBag,faTrash } from "@fortawesome/free-solid-svg-icons"
+const Cartcontext = createContext(false)
+function Cart({value}) {
+  const { clicked, setClicked } = useContext(ProductContxt);
+  const { cartState, setCart } = useContext(Cartcontext);
+  const { cartProudect, setCartProudect } = useContext(CartProudectContext)
+  let total = 0
+  const price = cartProudect.map((e) => e.price)
+  price.forEach(item=>total+=item)
+  let style={}
+  clicked?style={height:'380px'}:style={height:'0px'}
+  const Cartproudcts = cartProudect?.map((e) =>
```

Comment:
Create a sperate file for CartProduct component

Reply:
done